far/farlang.templ.m4

```diff
@@ -15496,7 +15496,7 @@ upd:"App Alias:"
 upd:"App Alias:"
 upd:"App Alias:"
 upd:"App Alias:"
-upd:"App Alias:"
+upd:"Alias aplikacji:"
```

Comment:
@BestiaPL FYI when the line is properly translated it's ok to remove the `upd:` prefix: it's used to indicate how many lines are not yet translated ([example](https://ci.appveyor.com/project/FarGroup/farmanager/builds/37223072/job/91cb2bccqciyh5qq?fullLog=true#L2920))